chrome/content/coniks/context.js

```diff
@@ -10,6 +16,9 @@ function ConiksContext(opts) {
     return null;
   }
   this.account = opts.account;
+  if (opts.fingerprint !== undefined || opts.fingerprint !== null) {
+    this.fingerprint = opts.fingerprint;
```

Comment:
This will be the public key fingerprint for contacts? But it seems like https://github.com/arlolra/ctypes-otr/pull/82/commits/0244fa5931b1e53f8eb5b0c27fee6ac9fe2e2220#diff-fdaf1540abbe9793707217dc6758ef60R52 sets the private key fingerprint for the account. It might be helpful to distinguish between the two more clearly.